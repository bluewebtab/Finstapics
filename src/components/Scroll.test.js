import {shallow, mount, render} from 'enzyme';
import React from 'react';
import Scroll from './Scroll';

it('expect to render Scroll component', () => {
    //shallow: tests components in isolation (excluding child components)
    expect(shallow(<Scroll />)).toMatchSnapshot();
})


