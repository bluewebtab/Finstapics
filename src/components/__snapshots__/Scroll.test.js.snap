// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `ShallowWrapper {}`;

exports[`expect to render Scroll component 1`] = `ShallowWrapper {}`;
